Tools/autotest/apmrover2.py

```diff
@@ -460,6 +460,74 @@ def test_servorelayevents(self):
             raise NotAchievedException()
         self.progress("Pin mask changed after relay command")
 
+    def get_message_rate(self, victim_message, timeout):
```

Comment:
those belong to common

Reply:
Will move the code up, if not the call to it.